api/app_sph_lms/models.py

```diff
@@ -257,7 +257,7 @@ class Course(models.Model):
             blank=True
         )
     author = models.ForeignKey(
-            Trainer,
+            User,
```

Comment:
please update Course model as a whole, refer to the new ERD.

P.S. 
Company field might cause conflicts, its okay to leave it for. Remove code, category, and preview_vid_path